locales/en-US/addon.properties

```diff
@@ -24,9 +24,11 @@ tour_title_intro= Tame Your Tabs!
 tour_title_collapse= The Space You Need
 tour_title_restore= Easy In, Easy Out
 dismiss_label= Not now
+dismiss_label_reminder_tour= Don't show this again
```

Comment:
Do apostrophes in properties work?  (The syntax highlighting is throwing me off…)